src/components/chipsInput/index.tsx

```diff
@@ -146,6 +147,24 @@ const ChipsInput = forwardRef((props: ChipsInputProps, refToForward: React.Ref<a
     );
   }, [chips, leadingAccessory, defaultChipProps, removeMarkedChip, markedForRemoval, maxChips]);
 
+  const isRequired = useMemo(() => {
+    if (!validate) {
+      return false;
+    }
+
+    const inputValidators = Array.isArray(validate) ? validate : [validate];
+    return inputValidators.includes('required');
+  }, [validate]);
+
+  const validateChipsInput = useCallback(() => {
```

Comment:
The only validity here is 'required' but the user might pass other validations, mainly a custom validation function

Reply:
Fixed, can be tested by adding the following to the screen:
```
validate={['required', () => {
    return this.state.chips2[0]?.label !== 'Bla';
  }]}
validateOnChange
```

The code handling this is a little complicated, but I'm not sure it can be simplified.